time beginTime
duration duration
float64 x
float64 y
float64 theta
